worlds/khrecom/docs/khrecom_en.md

```diff
@@ -0,0 +1,57 @@
+# Kingdom Hearts RECOM Randomizer Setup Guide
+
+##Setting up the required mods
+1. Install OpenKH and the LUA Backend.
+
+    Download the [latest release of OpenKH](https://github.com/OpenKH/OpenKh/releases/tag/latest)
+    
+    Extract the files to a directory of your choosing
+    
+    Open `OpenKh.Tools.ModsManager.exe` and run first time set up
+    
+    When prompted for game edition, choose PC Release via Epic Games Store and navigate to your `KH_1.5_2.5` folder in the path box and click `Next`
+    
+    When prompted, install Panacea, then click `Next`
+    
+    When prompted, check ReCoM and click `Install and configure LUA backend`, then click `Next`
+    
+    Ensure that `Launch via Epic Games` is checked and click `Next`
+    
+    You may skip extracting game data for ReCoM by checking `Skip Game Extraction` and clicking `Next`
+    
+    Click `Finish`.
+    
+2. Open `OpenKh.Tools.ModsManager.exe`
+
+3. Click the drop down menu at the top-right and choose `Re:Chain of Memories`
+
+4. Click `Mods>Install a New Mod`
+
+5. In `Add a new mod from GitHub` paste `gaithern/KH-RECOM-AP-LUA`
+
+6. Click Install
```

Comment:
```suggestion
6. Click `Install`
```
For consistency, since you put the button name in ticks elsewhere

Reply:
Applied, thanks!